chain/client/Cargo.toml

```diff
@@ -50,7 +50,6 @@ delay-detector = { path = "../../tools/delay_detector", optional = true }
 near-logger-utils = { path = "../../test-utils/logger" }
 near-actix-test-utils = { path = "../../test-utils/actix-test-utils" }
 testlib = { path = "../../test-utils/testlib" }
```

Comment:
In the next PR, I suggest tackling this dependency, so that `chain/client` becomes free of cyclic deps.